task_spec/spec/acceptance/init_spec.rb

```diff
@@ -69,7 +71,7 @@ def log_output_errors(result)
     # else
     # end
     case target_platform
-    when %r{osx-13}
+    when %r{el-9-aarch64}
```

Comment:
why is this changed / why do you remove osx13 here? That change isn't reflected in the PR title